src/ReflectionClosure.php

```diff
@@ -271,6 +272,9 @@ public function getCode()
                         case T_CURLY_OPEN:
                         case T_DOLLAR_OPEN_CURLY_BRACES:
                         case '{':
+                            if ($context === 'string') {
```

Comment:
This should be only used for T_DOLLAR_OPEN_CURLY_BRACES case.
Anyway, when is the 'string' state "resumed"?

Reply:
I think also should be use in T_CURLY_OPEN
like: `return "{$a[foo]} is $a[0]";`

Reply:
Probably